src/sso/pages/AppDetails.tsx

```diff
@@ -65,6 +71,11 @@ const AppDetails = () => {
     setDetails(app);
   }
 
+  const logout =  async () => {
+    if(user) await user.logout();
+    window.location.href = document.referrer;
```

Comment:
on direct in demo home.tsx we have check if account doesn't exists show alert account name not found and user.logout clears the account name. so i guess on logout we don't need to show any error.

Reply:
no, we should send it to the external website, they will decide weather to show the error